static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -52,6 +52,7 @@ const KNOWN_NUMERIC_FIELDS = new Set([
   SpanIndexedField.MESSAGING_MESSAGE_BODY_SIZE,
   SpanIndexedField.MESSAGING_MESSAGE_RECEIVE_LATENCY,
   SpanIndexedField.MESSAGING_MESSAGE_RETRY_COUNT,
+  SpanIndexedField.TRANSACTION_ID,
```

Comment:
Added this as a numeric field so that we don't show it as a tag option later in code when we filter out all the numeric fields

Reply:
maybe rename the set? `HIGH_CARDINALITY_TAGS`

Reply:
Renamed, thanks for suggestion